python/tests/adapters/local/test_local_llm_chat_model.py

```diff
@@ -21,15 +21,14 @@
 from beeai_framework.tools.weather.openmeteo import OpenMeteoTool
 
 
-@pytest.mark.skip(reason="This test downloads model from huggingface hub")
-class TestLocalChatModel:
+@pytest.mark.e2e
+@pytest.mark.asyncio
```

Comment:
I think this will apply the decorators to all methods which might not be desired, not sure about that.